app/src/main/java/com/github/libretube/ui/views/CustomExoPlayerView.kt

```diff
@@ -202,6 +205,22 @@ internal class CustomExoPlayerView(
             override fun onScrubStop(timeBar: TimeBar, position: Long, canceled: Boolean) {}
         })
 
+        setControllerVisibilityListener(
+            ControllerVisibilityListener { visibility ->
+                playerViewModel?.isFullscreen?.value?.let { isFullscreen ->
+                    if (!isFullscreen) return@let
+                    when (visibility) {
+                        View.VISIBLE -> {
+                            activity.showSystemBars(WindowInsetsCompat.Type.statusBars())
+                        }
+                        View.GONE -> {
+                            activity.hideSystemBars(WindowInsetsCompat.Type.statusBars())
```

Comment:
Same as above: Due to #3069, hiding the navigation bars here too might fix issues for some devices.